aqueduct/gateway/tests/utils/base.py

```diff
@@ -95,7 +100,7 @@ def setUpClass(cls):
                     f"vLLM integration tests require vllm to be installed. "
                     f"Import error: {_VLLM_IMPORT_ERROR}"
                 )
-            print(f"\nStarting vLLM server for {cls.__name__}...")
+            logger.info("\nStarting vLLM server for %s...", cls.__name__)
```

Comment:
Use prints for test related info to be able to differentiate application logs from test logs. Much easier to see and debug in CI etc!